python/ray/tune/scripts.py

```diff
@@ -28,9 +28,26 @@ def cli():
     default=None,
     type=str,
     help="Select filter in the format '<column> <operator> <value>'.")
-def list_trials(experiment_path, sort, output, filter_op):
+@click.option(
+    "--columns",
+    default=None,
+    type=str,
+    help="Select columns to be displayed.")
+@click.option(
+    "--result-columns",
+    "result_columns",
```

Comment:
supposed to have --?

Reply:
This is the convention used in Clicks 7.0 when the flag and corresponding variable name are different. Do we want to allow users to use `--result_columns` as a flag as well?